source/simulator/solver/stokes_matrix_free_local_smoothing.cc

```diff
@@ -1610,13 +1610,18 @@ namespace aspect
       sim.compute_initial_velocity_boundary_constraints(constraints_v);
       sim.compute_current_velocity_boundary_constraints(constraints_v);
 
-
       VectorTools::compute_no_normal_flux_constraints (dof_handler_v,
                                                        /* first_vector_component= */
                                                        0,
                                                        this->get_boundary_velocity_manager().get_tangential_boundary_velocity_indicators(),
                                                        constraints_v,
                                                        this->get_mapping());
+
+      sim.prescribed_solution_manager.constrain_solution(constraints_v);
+
+      // Let plugins add more constraints if they so choose:
+      sim.signals.post_constraints_creation(*this, constraints_v);
```

Comment:
Oh, so we had a bug here? Is that why the `prescribed_velocity` test and the other signal tests need an update even though they dont use the prescribed solution manager?